### PR TITLE
fix(manager): added validation and responsive chart

### PR DESCRIPTION
**Pull request by akhilmhdh**
## Explain the feature/fix

1. Added length validation for fields
2. Applied label to field names
3. Responsive charts
4. Clickable links in table
5. Added search in spa property list page

## Does this PR introduce a breaking change

No

## Screenshot(s)

<details>
<summary>View Screenshots</summary>

## SPA Property List Page

Screenshot 2022-09-16 at 4 06 18 PM

</details>

### Ready-for-merge Checklist

- [x] Expected files: all files in this pull request are related to one feature request or issue (no stragglers)?
- [x] Did you update or add any necessary documentation (README.md, WHY.md, etc.)?
- [x] Was this feature demo'd and the design review approved?

### CODE REVIEW

File: packages/manager/src/views/SPAPropertyDetailPage/SPAPropertyDetailPage.tsx [hunk 1]
```diff
@@ -118,14 +118,14 @@ export const SPAPropertyDetailPage = (): JSX.Element => {
           >
             <Split hasGutter className="pf-u-mt-md">
               <SplitItem isFilled>
-                <Card style={{ maxWidth: '100%' }} isFullHeight>
+                <Card isFullHeight style={{ height: '320px' }}>
```
Comment (hybridx) [inline packages/manager/src/views/SPAPropertyDetailPage/SPAPropertyDetailPage.tsx:121]: @akhilmhdh I think we can also try styled components

File: packages/manager/src/views/WebPropertyDetailPage/WebPropertyDetailPage.tsx [hunk 2]
```diff
@@ -138,80 +160,99 @@ export const WebPropertyDetailPage = (): JSX.Element => {
...
-                        </Td>
-                      </Tr>
-                    </Tbody>
-                  ))}
+                                </Thead>
+                                <Tbody>
+                                  {spaProperties?.data?.[identifier].map((prop) => (
+                                    <Tr key={`expandable-property${prop.id}`}>
+                                      <Td>
+                                        <Label color="gold" isCompact>
```
Comment (hybridx) [inline packages/manager/src/views/WebPropertyDetailPage/WebPropertyDetailPage.tsx:216]: @akhilmhdh we can think of a better color I suppose

**Review by hybridx — APPROVED** [review]
Few changes, other than that LGTM

**Review by arkaprovob — APPROVED** [review]

File: packages/manager/src/views/SPAPropertyDetailPage/SPAPropertyDetailPage.tsx (hunk 1 above)
Comment (akhilmhdh) [reply packages/manager/src/views/SPAPropertyDetailPage/SPAPropertyDetailPage.tsx:121]: @hybrdix We actually dont need styled component. Its used when u need to style a lot of things. For minor tweeks like custom height just use style props.

1. If its used everywhere like removing anchor text decoration write a utility css class in global file

2. Override patternfly css variable if its used everywhere. As we are using patternfly this is recommended most of the cases.

3. Use nextjs style syntax. (Least needed)

As we are using patternfly and some utility class of pfe introducing styled components is unnecessary bundle.

**Pull request merged**